data_acquisition/data_acquisition.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import os
+import psycopg2
+import getpass
+
+# for connecting to remote database
+# conn = psycopg2.connect(database='yourdb', user='dbuser', password='abcd1234', host='server', port='5432', sslmode='require')
+# for creating local database initially
+# cur.execute("CREATE TABLE Jobs(Id SERIAL PRIMARY KEY, Title TEXT, Description TEXT, Company TEXT, Location TEXT, SalaryMin INTEGER, PostURL TEXT)")
+# con.commit()
+
+# make ZipRecruiter API call
+API_KEY = os.getenv("ZIPRECRUITER_API_KEY")
```

Comment:
If `ZIPRECRUITER_API_KEY` is not present, this will throw a `KeyError` - should be relatively obvious what the problem is, but you might still consider `try`/`except`.

Reply:
looks like it'll default to None, I'll add a check for that.

Reply:
Oh right, I'm thinking of `os.environ`. Still, good to add a check!